libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts

```diff
@@ -768,4 +779,8 @@ export class AdminClientsService {
       ),
     )
   }
+
+  private isUserSuperAdmin = (user: User) => {
```

Comment:
~ I would rename the function to better match the `AdminPortalScope.idsAdminSuperUser`

```suggestion
  private isSuperUser = (user: User) => {
```